LICENSE.md

```diff
@@ -29,18 +34,25 @@ Contributions may come in any form, and include (but are not limited to):
  * commentary
  * example code
 
-If you do not want your work and ideas to become incorporated into Gno, do not
-make contributions to this project; or keep such work and ideas secret. The
-creation of contributions that may in the future become known to this project's
-owners and maintainers constitutes a willing contribution to this project in
-accordance with this agreement.
-
-You may keep any number of copies of this project for your personal purposes as
-long as it isn't for any for-profit, commercial, religious, government, NGO, or
-distributed ledger (blockchain) purposes.
-
-THE WORKS ARE PROVIDED WITHOUT WARRANTY OF CORRECTNESS. IN NO EVENT SHALL THE
-AUTHORS, COPYRIGHT HOLDERS, OWNERS, OR MAINTAINERS BE LIABLE FOR ANY CLAIM,
-DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
-OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE WORK OR THE USE OR
-OTHER DEALINGS IN THE WORK.
+If you do not want your contribution to become incorporated into this project,
+do not make contributions to this project. The creation of contributions that
+may in the future become known to this project's owner and maintainer
+constitutes a willing contribution to this project in accordance with this
+license.
+
+You may keep a copy of this project only for your personal, commercial, and
+non-commercial purposes as long as such purposes are not associated with any
+religious or distributed ledger (blockchain) subject matter.
```

Comment:
NOTE: commercial, government, and NGO usage is allowed, as long as it doesn't involve blockchains.

Reply:
The intent is to restrict the usage of gno IP for launching new blockchain projects, until gno.land itself is first established, and inter-blockchain license terms are also established.